src/App.scss

```diff
@@ -2,36 +2,123 @@ iframe {
   display: none;
 }
 
+.title {
+  font-size: 50px;
+}
+
+.containerForApp {
+  width: 100%;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.App {
+  border: 2px black solid;
+  display: flex;
+  flex-direction: column;
+  max-width: 75%;
+  align-items: center;
+}
+
+.formUsers {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  gap: 20px;
+
+  .field {
+    font-family: "Franklin Gothic Medium", Arial, sans-serif;
+    font-size: 30px;
+  
+    &-form {
+      display: block;
+      width: 100%;
+      background-color: aqua;
+      padding: 8px 16px;
+      line-height: 25px;
+      font-size: 14px;
+      font-weight: 500;
+      font-family: inherit;
+      border-radius: 6px;
+      -webkit-appearance: none;
+      border: 1px solid var(--input-border);
+      transition: border 0.3s ease;
+
+      &::placeholder {
+        color: var(--input-placeholder);
+      }
+
+      &:focus {
+        outline: none;
+        border-color: var(--input-border-focus);
+      }
+    }
+  }
+}
+
+.lableTitle {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+}
+
+.labelUser {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+}
+
+.submit-button {
+  width: 200px;
+  appearance: button;
+  background-color: #1652f0;
+  border: 1px solid #1652f0;
+  border-radius: 4px;
+  box-sizing: border-box;
+  color: #fff;
+  cursor: pointer;
+  font-size: 14px;
+  line-height: 1.15;
+  overflow: visible;
+  padding: 12px 16px;
+  position: relative;
+  text-align: center;
+  text-transform: none;
+  transition: all 80ms ease-in-out;
+  user-select: none;
+  -webkit-user-select: none;
+  touch-action: manipulation;
+}
+
+.selectorUsers {
+  width: 200px;
+  height: 36px;
+  border-radius: 5px;
+  border: none;
+  background: aqua;
+}
+
 html {
-  font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;
+  margin: 0 auto;
+  background: linear-gradient(90deg, rgb(187, 70, 197),
+  rgb(48, 233, 177) 49%,
+  rgb(42, 202, 234) 100%);
```

Comment:
Avoid using inline styles. Consider creating CSS variables for the gradient colors and use them here.